app/routes/events/components/EventDetail/EventDetail.css

```diff
@@ -48,6 +48,13 @@
   align-items: center;
 }
 
+.iconWithTextMazeMap {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  flex-wrap: wrap;
+}
```

Comment:
This is just `<Flex wrap alignItems="center" gap={20}  />`

Reply:
True